tests/jax/test_custom_call_compute.py

```diff
@@ -0,0 +1,575 @@
+# Copyright (c) 2022-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
Since we have a clear separation of the 2 frameworks in TE, I think it also makes sense in this PR to add the PyTorch tests in a separate folder `tests/pytorch`. The ONNX stuff can be moved there as well. So the tests would have 2 directories (`jax` and `pytorch`) and no files.

Reply:
We'll also need to adjust paths at https://github.com/jeng1220/TransformerEngine/blob/rjeng/init_jax/qa/L0_unittest/test.sh.

Reply:
I can help to submit another PR to move files regarding PyTorch into `tests/pytorch`. Hope to focus on TE/JAX in this PR

Reply:
@ksivaman and @timmoon10 ,
It should be solved by https://github.com/NVIDIA/TransformerEngine/pull/78